### ISSUE TITLE
JSNM - Data Entry - Part Name Issue

### ISSUE THREAD

**Issue by cefilipek** [ArctosDB/data-migration#190]
I am having one problem with the data entry. I see that fossil is in the code table, but all that comes up when I type fossil is "shell (fossil)" and "trace fossil", neither of which describe my object. Do different collections types have access to different words in the code table?
![Part Names](https://user-images.githubusercontent.com/56844074/72748982-5ff48580-3b7e-11ea-8ab9-f93438dfca46.png)
![Code Table Fossil](https://user-images.githubusercontent.com/56844074/72748993-6551d000-3b7e-11ea-8c0a-00bb195d6ba2.png)

**Comment by Jegelewicz** [comment]
We have mostly abandoned using "fossil" as a part name. What exactly do you have? For instance, an ammonite would be a shell, the taxonomy would indicate it was a fossil species, but you could also add the [part preservation attribute](http://arctos.database.museum/info/ctDocumentation.cfm?table=CTPART_PRESERVATION) with the value "fossil" or any of the other "fossil" types that apply.

**Comment by Jegelewicz** [comment]
> Do different collections types have access to different words in the code table?

Yes, the codes in the collection type column tell you which collections have access to a given part. This is primarily to keep the choices available in check when doing data entry.

**Comment by cefilipek** [comment]
That makes sense. I didn't realize that Arctos had moved away from that. It is a Macroneuropteris Scheuchzeri, identified as a Neuropteris Decipens by the collector, so it is a type of seed plant

**Comment by Jegelewicz** [comment]
Plants are tough! Can you look at the specimen? If so, you could be more precise as to what plant parts are included (leaf, branch, root, seed, etc). If time doesn't allow that, I would use "whole organism" and enter a part remark to let everyone know that you have not verified that the entire plant is preserved.

**Comment by cefilipek** [comment]
It looks like it's just the leaf to me, so would I choose specifically leaf in this case?

**Comment by Jegelewicz** [comment]
Yep!

**Comment by sharpphyl** [comment]
We prefer to continue to use "shell (fossil)" as a part name.  Some extant mollusk species also appear in the fossil record.  The fact that they have geology would indicate a fossil specimen, but I like also using the fossil part name to make sure no one thinks we geolocated a marine species to the middle of today's Florida.

**Comment by Jegelewicz** [comment]
> shell (fossil)

Eventually this part name will be converted to "shell" with the preservation "fossil". See https://github.com/ArctosDB/arctos/issues/1460

The data structure exists to do this now, but it has been held back awaiting PostGreSQL.
